nrf_security/configs/nrf-config.h.template

```diff
@@ -8,22 +8,9 @@
  *  memory footprint.
  */
 /*
- *  Copyright (C) 2006-2018, ARM Limited, All Rights Reserved
- *  SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) 2001-2019, Arm Limited and Contributors. All rights reserved.
```

Comment:
Just wondering, did they timewarp back to 2001 in order to create this file, since it used to be copyrighted in since 2006 :thinking:

Reply:
This is based on updating towards the mbed TLS v2.23.0 config.h file. Keeping it as is to give a smaller diff in the future when this is merged with incoming changes